cmd/testhelpers_test.go

```
@@ -0,0 +1,53 @@
+}
+
+func captureStdout(t *testing.T, fn func()) string {
+	t.Helper()
+	r, w, err := os.Pipe()
+	if err != nil {
+		t.Fatal(err)
+	}
+	old := os.Stdout
+	os.Stdout = w
+	fn()
+	w.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `w.Close` is not checked

(errcheck)